cqf-fhir-utility/src/main/java/org/opencds/cqf/fhir/utility/repository/InMemoryFhirRepository.java

```diff
@@ -195,6 +198,28 @@ public <B extends IBaseBundle> B transaction(B transaction, Map<String, String>
         throw new NotImplementedException("The transaction operation is not currently supported");
     }
 
+    public static Bundle transactionStub(Bundle transaction, Repository repository) {
```

Comment:
This is R4 specific logic and cannot be in here.  This either needs to be done in a version agnostic way, (look to the BundleHelper) or with a factory that returns the version specific class based on the version of the Repository.